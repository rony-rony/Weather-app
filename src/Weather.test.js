import React from 'react';
import Weather from './components/Weather';
import { shallow } from 'enzyme';

it('renders Weather wrapper without crashing', () => {
    const wrapper = shallow(<Weather />);
    expect(wrapper).toMatchSnapshot();
  });
