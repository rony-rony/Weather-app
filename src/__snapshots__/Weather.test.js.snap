// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Weather wrapper without crashing 1`] = `ShallowWrapper {}`;
